app/components/card-breakdown-chart.js

```diff
@@ -0,0 +1,56 @@
+import Ember from 'ember';
+import layout from '../templates/components/card-breakdown-chart';
+
+export default Ember.Component.extend({
+  layout: layout,
+
+  bars: function () {
+    var deck = this.get('deck');
+    var cardGroups = deck.get('cardGroups');
+    var mainCount = deck.get('mainCount');
+    var stats = {
```

Comment:
What do you think about generating these based on the `card.mainType`s that are present in the deck? One absence I can think of is Plainswalker. [Looks like](http://mtg.wikia.com/wiki/Card_Types) Tribal might be another type that could show up. Not really sure if that actually ever is used as a first class type.